proto/asgt/v2alpha/type/training_info.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package asgt.v2alpha.type;
+
+option go_package = "github.com/e-conomic/vmlapis/gen/go/asgt/v2alpha/type;asgttype";
+
+import "google/protobuf/timestamp.proto";
+
+message TrainingInfo {
+    google.protobuf.Timestamp started_at = 1;
+    google.protobuf.Timestamp ended_at = 2;
+    string training_status = 3;
+}
```

Comment:
I also think that we could consider having a `repeated TrainingInfo` here, since, for example, on the consumer portal we show the last 5 trainings (and their details). I'd imagine that we'd want to have an endpoint like this on the api as well - so we'd return the last 5 trainings. Or we could keep this one as it is but add another endpoint that would return the last X trainings - the X would either be specified in the request or we can just decide on an arbitrary number.

Reply:
added the  training status message and repeated trainings